Felienne/hedy
# [UI] Add searching to the ACE editor

Enables searching in the ACE editor.

Fixes #3342 

**How to test**
Make sure to click the ACE editor and use the **ctrl+f** shortcut. This should open up the search bar. 
You can now search your code!

![image](https://user-images.githubusercontent.com/48225550/196524590-13ba307d-f720-4b1a-bf34-2d5f5bba34e8.png)
